app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```

Comment:
@CollinsTatang  Hi, i need some clarification on creating a `new` action for comments too. I am using the new comment form as a partial in `posts/index` page and not serving the form in a new html page. wouldn't `create` action be sufficient for this.

Reply:
Hi @ram1117
It could be sufficient, but according to the project requirement, you are expected to create a separate controller action for create and new method.